onnxruntime/core/providers/webgpu/webgpu_kernel.h

```diff
@@ -23,9 +23,7 @@ class WebGpuKernel : public OpKernel {
   Status Compute(OpKernelContext* p_op_kernel_context) const override {
     ComputeContext context{*p_op_kernel_context};
 
-    context.PushErrorScope();
     Status s = ComputeInternal(context);
-    ORT_RETURN_IF_ERROR(context.PopErrorScope());
```

Comment:
can we keep them for `context_.ValidationMode() >= ValidationMode::Full`?

Reply:
This helps to quickly identify which kernel causes validation error

Reply:
Makes sense, Done！

Reply:
Ping @fs-eire